instance/prod_config.py.default

```diff
@@ -32,11 +32,12 @@ MAPLIST = [
     'de_train',
     'de_dust2',
     'de_season',
+    'de_vertigo',
```

Comment:
Can this be ordered so the first 7 entries are the same as in DEFAULT_MAPLIST, then it's the remaining ones after?